### ISSUE TITLE
make sure page indexing doesn't slow down the page

### ISSUE THREAD

**Issue by afeld** [18F/hub#263]
e.g. using a [Web Worker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API).

https://github.com/mattpodwysocki/lunr.js-example

**Comment by mbland** [comment]
Curious: If we use a WebWorker, that requires a recent browser, correct? Would older browsers be at a distinct disadvantage, then?

**Comment by afeld** [comment]
Yes. There are other tricks like serializing the index in localStorage etc., though that would be tricky (if not impossible) in old browsers too.
